src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -347,7 +347,10 @@ public record InstanceConfig
     IDictionary<Endpoint, ApiAccessRule>? ApiAccessRules = null,
     IDictionary<PrincipalId, GroupId[]>? GroupMembership = null,
     IDictionary<string, string>? VmTags = null,
-    IDictionary<string, string>? VmssTags = null
+    IDictionary<string, string>? VmssTags = null,
+    string? Authority = "",
```

Comment:
> string? [](http://example.com/codeflow?start=3&length=8)

Do we need nullable types here? Don't we have required parameters?

Reply:
Removed.